### PR TITLE
Expose the ETag header in 304 responses.

### PR DESCRIPTION
**Pull request by Natim**
Add a test to show the behavior in https://github.com/mozilla-services/cliquet/issues/631

### CODE REVIEW

**Comment by leplatrem** [comment]
See https://github.com/mozilla-services/cliquet/issues/631#issuecomment-176761484

**Comment by leplatrem** [comment]
@almet r?

**Comment by almet** [comment]
That looks good to me!

**Comment by almet** [comment]
r+

File: kinto/plugins/default_bucket/__init__.py [hunk 1]
```diff
@@ -120,7 +120,8 @@ def default_bucket(request):
     try:
         response = request.invoke_subrequest(subrequest)
     except httpexceptions.HTTPException as error:
-        if error.content_type == 'application/json':
+        is_redirect = error.status_code < 400
```
Comment (Natim) [inline kinto/plugins/default_bucket/__init__.py]: I would have tested 304 only there.

**Pull request merged**